understanding/21/orientation.html

```diff
@@ -62,7 +62,7 @@ <h2>Examples</h2>
 				<li><strong>Example 4: Check deposit in banking app</strong><br>
 				An example where orientation is essential could be a banking app that requires the device be in landscape mode to easily and accurately capture an image of a check for deposit. These paper forms are typically about twice as wide as they are high.</li>
 				<li><strong>Example 5: Piano app</strong><br>
-				An example where orientation is essential could be a piano app that requires the device to be in landscape mode to allow room for enough of the piano keys to be functionally usable. Since a piano app is emulating a physical piano keyboard that needs to retain relative physical characteristics between keys, either too few keys would be available, or the keys would be much too narrow.</li>
+				An example where orientation is essential could be a piano app that requires the device to be in landscape mode allowing the piano keys to have enough room to be functionally usable. Since a piano app is emulating a physical piano keyboard that needs to retain relative physical characteristics between keys, either too few keys would be available, or the keys would be much too narrow.</li>
```

Comment:
```suggestion
				An example where orientation is essential could be a piano app that requires the device to be in landscape mode, allowing the piano keys to have enough room to be functionally usable. Since a piano app is emulating a physical piano keyboard that needs to retain relative physical characteristics between keys, either too few keys would be available, or the keys would be much too narrow.</li>
```